src/coreclr/tools/Common/Compiler/Win32Resources/ResourceData.UpdateResourceDataModel.cs

```diff
@@ -90,5 +98,16 @@ private byte[] FindResourceInternal(object name, object type, ushort language)
 
             return (byte[])resLanguage.DataEntry.Clone();
         }
+
+        private static string ToUpperForResource(string str)
+        {
+            // Undocumented semantic for Win32 Resource APIs
+            StringBuilder builder = new(str.Length);
+            foreach (char c in str)
+            {
+                builder.Append('a' <= c && c <= 'z' ? char.ToUpper(c) : c);
```

Comment:
Use invariant culture?

Reply:
Not in this case. That isn't how it is defined. We can chat about this offline if you'd like.